server/lib/poller.js

```diff
@@ -114,6 +141,34 @@ class Poller
             log.error(`when polling accounts at startup: ${err.message}`);
         }
     }
+
+    async manageCredentialErrors(access, err) {
+        try {
+            if (err.errCode) {
```

Comment:
To reduce indentation, can you do:

```
if (!err.errCode)
  return;
```

Then the rest of the function can have one less indent!